packages/glsp-sprotty/src/types.ts

```diff
@@ -48,5 +48,7 @@ export const TYPES = {
     IDiagramOptions: Symbol('IDiagramOptions'),
     IDiagramStartup: Symbol('IDiagramStartup'),
     IToolManager: Symbol('IToolManager'),
-    IContributionProvider: Symbol('IContributionProvider')
+    IContributionProvider: Symbol('IContributionProvider'),
+    IContributionInitializer: Symbol('IContributionInitializer'),
+    IServiceProvider: Symbol('IServiceProvider')
```

Comment:
I don't think the two new symbols are used anywhere, probably a leftover of some intermediate changes?
I believe even the `IContributionProvider` could be removed.